src/Diagnostics.ModelsAndUtils/Models/RuntimeContext.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Diagnostics.ModelsAndUtils.Models
+{
+    public class RuntimeContext<TResource>
```

Comment:
You are right, thought of that as I was creating it and thought it might be being used in Scripts project, I see now that it is not. I will put it in RuntimeHost project.